packages/components/src/interfaces/internal/Wizgov.ts

```diff
@@ -0,0 +1,13 @@
+import type { IsomerSiteProps } from "~/types"
+
+export interface WizgovProps {
+  "data-agency": string
+}
```

Comment:
where did we get this prop from?

Reply:
can refer to here: https://github.com/opengovsg/wizgov

FYI checked with Ziyang from AskGov that no longer require CSS even though the docs say so